### PR TITLE
Issue #3216: Contact form categories: recipients should be rendered a…

### PR DESCRIPTION
**Pull request by opi**
…s a list

### CODE REVIEW

File: core/modules/contact/contact.admin.inc [hunk 1]
```diff
@@ -101,8 +101,10 @@ function contact_category_list($form, $form_state) {
     $form['categories'][$cid]['name'] = array(
       '#markup' => check_plain($cat['category']),
     );
+    $recipients = explode(',', check_plain($cat['recipients']));
+    $recipients_list = '<ul><li>' . implode('</li><li>', $recipients) . '</li></ul>';
     $form['categories'][$cid]['recipients'] = array(
-      '#markup' => check_plain($cat['recipients']),
+      '#markup' => $recipients_list,
```
Comment (klonos) [inline core/modules/contact/contact.admin.inc]: wondering if using `theme('item_list', $recipients);` is more appropriate here. Other than that, RTBC from me 👍

Comment (opi) [reply core/modules/contact/contact.admin.inc]: indeed, you're right. I always have this too-much-left-padding feeling with `item-list`, but it's the drup^backdrop way of printing a list of elements.

**Comment by opi** [comment]
reload tests

**Review by klonos — APPROVED** [review]
👍 RTBC

**Comment by quicksketch** [comment]
Thanks @opi and @klonos! Merged into 1.x.

**Pull request merged**